mmdet/core/hook/sync_random_size_hook.py

```diff
@@ -17,26 +18,33 @@ class SyncRandomSizeHook(Hook):
             by 32, and then change the dataset output image size.
             Default: (14, 26).
         img_scale (tuple[int]): Size of input image. Default: (640, 640).
-        interval (int): The interval of change image size. Default: 10.
+        interval (int): The epoch interval of change image size. Default: 1.
         device (torch.device | str): device for returned tensors.
             Default: 'cuda'.
     """
 
     def __init__(self,
                  ratio_range=(14, 26),
                  img_scale=(640, 640),
-                 interval=10,
+                 interval=1,
                  device='cuda'):
+        warnings.warn('DeprecationWarning: SyncRandomSizeHook is deprecated. '
+                      'Please use Resize class to achieve similar functions. '
```

Comment:
Resize class -> Resize pipeline. Also need to indicate the specific replacement method. Such as giving an example pipeline:```dict(
        type='Resize',
        img_scale=[(448, 448), (832, 832)],
        multiscale_mode='range',
        keep_ratio=True),```